setup.py

```diff
@@ -27,15 +33,22 @@ def get_readme():
     packages=find_packages(),
     entry_points={
         'console_scripts': [
-            'argparse-manpage=build_manpages.cli:main',
+            'argparse-manpage=argparse_manpage.cli:main',
         ],
     },
     description='Build manual page from python\'s ArgumentParser object.',
     long_description=get_readme(),
     long_description_content_type='text/markdown',
     cmdclass={
         'build_manpages': build_manpages,
-        'build_py': get_build_py_cmd(build_py),
-        'install': get_install_cmd(install),
+        'build_py': get_build_py_cmd(),
+        'install': get_install_cmd(),
     },
+    # TODO: This is only needed by 'build_manpages' module, not
+    # 'argparse_manpage'.  Switch to extras_require that will only affect
+    # the 'build_manpages' sub-package.  We need to drop EL7/Python 2.7
+    # support first.
+    install_requires=[
+        'setuptools',
+    ],
```

Comment:
Going to use this.  I bet it is worth it.

Reply:
If you do that,you can drop all the try-except imports.

Reply:
Honestly, if I was looking for a temporary thing, I'd just add the 1 line from https://github.com/praiskup/argparse-manpage/issues/63#issuecomment-1287103852 instead of changing 15 files very much. But if you want to mix a temporary workaround with refactoring, that' your call :)

Reply:
I agree with everything you said ... I'm hitting my Python limits in this refactor;  which is worth doing sooner or later.

Now that the optional dep problem turns out to be more complicated than I'm able to handle in the next release, the dilemma is whether to revert everything or keep it for the future release.